rust/src/lib/ifaces/mod.rs

```diff
@@ -20,42 +21,45 @@ mod ovs;
 mod sriov;
 mod vlan;
 
-pub use base::*;
-pub use bond::{
+pub use self::base::*;
+pub use self::bond::{
     BondAdSelect, BondAllPortsActive, BondArpAllTargets, BondArpValidate,
     BondConfig, BondFailOverMac, BondInterface, BondLacpRate, BondMode,
     BondOptions, BondPortConfig, BondPrimaryReselect, BondXmitHashPolicy,
 };
-pub use bridge_vlan::{
+pub use self::bridge_vlan::{
     BridgePortTrunkTag, BridgePortVlanConfig, BridgePortVlanMode,
     BridgePortVlanRange,
 };
-pub use dummy::DummyInterface;
-pub use ethernet::{
+pub use self::dummy::DummyInterface;
```

Comment:
Use the relative path `use self::` warrants another commit ?

Reply:
Removed unrelated changes